examples/v1beta1/kubeflow-training-operator/trainjob-pytorch.yaml

```diff
@@ -0,0 +1,65 @@
+---
+apiVersion: kubeflow.org/v1beta1
+kind: Experiment
+metadata:
+  namespace: kubeflow
+  name: torch-distributed-example
+spec:
+  parallelTrialCount: 3
+  maxTrialCount: 12
+  maxFailedTrialCount: 3
+  objective:
+    type: minimize
+    goal: 0.001
+    objectiveMetricName: loss
+  algorithm:
+    algorithmName: random
+  parameters:
+    - name: lr
+      parameterType: double
+      feasibleSpace:
+        min: "0.01"
+        max: "0.05"
+    - name: momentum
+      parameterType: double
+      feasibleSpace:
+        min: "0.5"
+        max: "0.9"
+    #- name: arg
+    #  parameterType: int
+    #  feasibleSpace:
+    #    min: 1
+    #    max: 5
+  trialTemplate:
+    primaryContainerName: pytorch
```

Comment:
This should be `node`, right ?

Reply:
Not quite understand your question
This is copy from pytorchjob-mnist.yaml and I keep it the same

Reply:
I have done it. Please state in somewhere in the doc that this field should be referenced to the ClusterTrainingRuntime. Since I thought it is a custom defined name of the container by the user